modules/libcom/src/osi/os/WIN32/osdFindSymbol.c

```diff
@@ -28,19 +47,46 @@ epicsShareFunc void * epicsLoadLibrary(const char *name)
 
 epicsShareFunc const char *epicsLoadError(void)
 {
-    static char buffer[100];
+    STORE char buffer[100];
+    DWORD n;
 
-    FormatMessage(
+    n = FormatMessage(
         FORMAT_MESSAGE_FROM_SYSTEM,
         NULL,
         epicsLoadErrorCode,
         0,
         buffer,
         sizeof(buffer)-1, NULL );
```

Comment:
You should also pass FORMAT_MESSAGE_IGNORE_INSERTS to the function in this situation as you are not supplying any arguments the message might require.